### ISSUE TITLE
No rule to make target [Ubuntu 18.04] - Erro ao compilar projeto

### ISSUE THREAD

**Issue by wendersonj** [A-Ribeiro/OpenGLStarter#1]
Erro ao compilar projeto no CodeBlocks. Código dando erro para encontrar a regra _target_ na hora de realizar o _maker_ dos arquivos com o seguinte problema crítico:

>  No rule to make target `/usr/lib/x86_64-linux-gnu/libGL.so'

Solução:

```
$ sudo rm /usr/lib/x86_64-linux-gnu/libGL.so 
$ sudo ln -s /usr/lib/libGL.so.1 /usr/lib/x86_64-linux-gnu/libGL.so 

```
[Fonte](http://techtidings.blogspot.com/2012/01/problem-with-libglso-on-64-bit-ubuntu.html)

**Comment by wendersonj** [comment]
Caso o link da fonte saia do ar, disponibilizo seu conteúdo e o respectivo autor.

`
Saturday, January 14, 2012
Problem with libGL.so on 64-bit Ubuntu

I have an Nvidia card and as expected I installed Nvidia drivers for my ubuntu. But, whenever I compiled programs that used OpenGL ( libGL.so ) I ran into compile errors similar to:

No rule to make target `/usr/lib/x86_64-linux-gnu/libGL.so'

I was sure there was some problem with my libGL.so file so I ended up looking at it properties and found out that it was a broken link pointing to 'mesa/libGL.so' which in turn was pointing to 'libGL.so.1' which never existed.

bpaluri3@bpaluri3:~/libfreenect/build$ ls -l /usr/lib/x86_64-linux-gnu/libGL.so 
lrwxrwxrwx 1 root root 13 2011-08-10 04:20 /usr/lib/x86_64-linux-gnu/libGL.so -> mesa/libGL.so

bpaluri3@bpaluri3:~/libfreenect/build$ ls -l /usr/lib/x86_64-linux-gnu/mesa/libGL.so 
lrwxrwxrwx 1 root root 10 2011-08-10 04:20 /usr/lib/x86_64-linux-gnu/mesa/libGL.so -> libGL.so.1


Then I verified the libGL.so.1 in my /usr/lib directory and found out that it pointed to libGL.so.290.10 which is the file provided by the nvidia driver.

bpaluri3@bpaluri3:~/libfreenect/build$ ls -l /usr/lib/libGL.so.1 
lrwxrwxrwx 1 root root 15 2012-01-09 13:53 /usr/lib/libGL.so.1 -> libGL.so.290.10

I just overwrote the symbolic link in `/usr/lib/x86_64-linux-gnu/libGL.so' with the nvidia openGL library i.e. `/usr/lib/libGL.so' ( I had to delete the old symbolic link before I do this ).

```
bpaluri3@bpaluri3:~/libfreenect/build$ sudo rm /usr/lib/x86_64-linux-gnu/libGL.so 
bpaluri3@bpaluri3:~/libfreenect/build$ sudo ln -s /usr/lib/libGL.so.1 /usr/lib/x86_64-linux-gnu/libGL.so 

```
The final two statements are the fixes for the libGL error and I hope anyone who runs into a similar issue will find this information useful.

Posted by Manohar Paluri at 8:51 AM
`
